From 1b9dde4fafa8663fe46b315aa0f44f0e18addd48 Mon Sep 17 00:00:00 2001
From: chrisw2 <chrisw2@rpi01>
Date: Wed, 11 Jul 2012 00:19:00 +0100
Subject: [PATCH] Change to add SCSI tape support as loadable modules to
 kernel config after line 264 add these two lines:
 CONFIG_CHR_DEV_ST=m CONFIG_CHR_DEV_OSST=m


diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 0820055..211d4ff 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -262,6 +262,8 @@ CONFIG_MISC_DEVICES=y
 CONFIG_SCSI=y
 # CONFIG_SCSI_PROC_FS is not set
 CONFIG_BLK_DEV_SD=y
+CONFIG_CHR_DEV_ST=m
+CONFIG_CHR_DEV_OSST=m
 CONFIG_BLK_DEV_SR=m
 CONFIG_SCSI_MULTI_LUN=y
 # CONFIG_SCSI_LOWLEVEL is not set
